### PR TITLE
Revendor g/g v1.97

### PR DESCRIPTION
**Pull request by ftl**
## References

Preparation for #29 

## Additional Description

None

## Release Notes

None

### CODE REVIEW

File: pkg/controller/audit/actuator.go [hunk 1]
```diff
@@ -560,7 +561,7 @@ func seedObjects(auditConfig *v1alpha1.AuditConfig, secrets map[string]*corev1.S
 				},
 			},
 			Spec: policyv1.PodDisruptionBudgetSpec{
-				MinAvailable: utils.IntStrPtrFromInt32(1),
+				MinAvailable: ptr.To(intstr.FromInt32(1)),
```
Comment (Gerrit91) [inline pkg/controller/audit/actuator.go:564]: Historically, we have an own pointer package. :)

```suggestion
				MinAvailable: pointer.Pointer(intstr.FromInt32(1)),
```

**Review by Gerrit91 — COMMENTED** [review]
Nice. We're on g/g 1.94 with the next metal-stack release. Do you think there will be any issues using this version of Gardener with this extension already being on 1.97?

**Comment by ftl** [comment]
> Nice. We're on g/g 1.94 with the next metal-stack release. Do you think there will be any issues using this version of Gardener with this extension already being on 1.97?

There is a breaking change in the ControllerDeployment. Gardener v1.96 switched from v1beta1 to v1. This might probably be a problem.

**Review by Gerrit91 — APPROVED** [review]
This works already on g/g 1.96.

**Pull request merged**